manifests/upstream/sealed-secrets/crd/kustomization.yaml

```diff
@@ -1,2 +1,2 @@
 resources:
-  - https://raw.githubusercontent.com/bitnami-labs/sealed-secrets/v0.17.1/helm/sealed-secrets/crds/sealedsecret-crd.yaml
\ No newline at end of file
+  - https://raw.githubusercontent.com/bitnami-labs/sealed-secrets/v0.17.1/helm/sealed-secrets/crds/sealedsecret-crd.yaml
```

Comment:
Set the version to the same as the base version (0.19.5?) and then also make crds/sealed-secrets.yaml